src/test/javascript/InstructorFeedbackEditJsTest.js

```diff
@@ -1,17 +0,0 @@
-QUnit.module('instructorFeedbackEdit.js');
-
-QUnit.test('getQuestionIdSuffix(questionNumber)', function(assert) {
-    var max = 9007199254740991; // Number.MAX_SAFE_INTEGER
-    var min = -max;
```

Comment:
Technical note: looking at this, `min` was probably 1 off from the actual minimum value, but it's good that we can get rid of this